test/spec/helper.distinct.js

```diff
@@ -6,7 +6,10 @@ var forEach = require('lodash/collection/forEach');
 var algoliasearchHelper = require('../../index.js');
 
 test('Distinct not set', function(t) {
```

Comment:
I do not understand why we add some more code the the helper distinct test while the PR is about throwing when unknown facet?